src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/resource/VisitResourceIntTest.kt

```diff
@@ -41,30 +59,134 @@ class VisitResourceIntTest : IntegrationTestBase() {
     @Test
     fun `get visit by prisoner ID`() {
 
-      webTestClient.get().uri("/visits/prisoner/$testPrisonerId")
-        .headers(setAuthorisation(roles = listOf("ROLE_PLACEHOLDER_VISIT")))
+      webTestClient.get().uri("/visits?prisonerId=FF0000AA")
+        .headers(setAuthorisation(roles = listOf("ROLE_VISIT_SCHEDULER")))
         .exchange()
         .expectStatus().isOk
         .expectBody()
         .jsonPath("$.length()").isEqualTo(1)
-        .jsonPath("$[-1].prisonerId").isEqualTo(testPrisonerId)
-        .jsonPath("$[-1].visitDateTime").isEqualTo(visitTime.toString())
+        .jsonPath("$[-1].prisonerId").isEqualTo("FF0000AA")
+        .jsonPath("$[-1].startTimestamp").isEqualTo(visitTime.toString())
         .jsonPath("$[-1].id").exists()
```

Comment:
curious what does the "-1" refer to in `$[-1]` ?

Reply:
negative values will select from the end of the array -1 = last, -2 second to last etc.
Not really useful in this test -  more a copy and paste from a previous.  I'll remove in next commit